src/components/tasks/NewTaskDialog.vue

```diff
@@ -0,0 +1,291 @@
+<template>
+  <v-dialog :value="value" width="600px" @input="$emit('input', false)">
+    <v-card class="pa-4">
+      <h2>New Task</h2>
+      <validation-observer ref="form" v-slot="{ invalid, handleSubmit }">
+        <form @submit.prevent="handleSubmit(publishTask)">
+          <validation-provider
+            v-slot="{ errors }"
+            rules="required|alpha_spaces|max:30"
+            name="title"
+          >
+            <v-text-field
+              v-model="title"
+              label="Title"
+              :error-messages="errors"
+            />
+          </validation-provider>
+          <validation-provider
+            v-slot="{ errors }"
+            rules="requiredSelect"
+            name="local group"
+          >
+            <v-select
+              v-model="localGroup"
+              :items="localGroups"
+              item-value="value"
+              item-text="text"
+              return-object
+              label="Local group"
+              :error-messages="errors"
+            />
+          </validation-provider>
+          <validation-provider
+            v-slot="{ errors }"
+            rules="requiredSelect"
+            name="working group"
+          >
+            <v-select
+              v-model="workingGroup"
+              :items="workingGroups"
+              item-value="value"
+              item-text="text"
+              return-object
+              label="Working group"
+              :error-messages="errors"
+            />
+          </validation-provider>
+          <p style="color: gray" class="caption">
+            Responsibilities
+          </p>
+          <validation-provider
+            v-slot="{ errors }"
+            rules="max:1000"
+            name="requirements"
+          >
+            <v-textarea
+              v-model="requirements"
+              label="Requirements (optional)"
+              placeholder="Skills, experience, equipment"
+              :error-messages="errors"
+            />
+          </validation-provider>
+          <validation-provider
+            v-slot="{ errors }"
+            rules="max:1000"
+            name="description"
+          >
+            <v-textarea
+              v-model="description"
+              label="Description (optional)"
```

Comment:
description should be required (see previous comment)
```suggestion
              label="Description"
```